client/src/constants/routing.js

```diff
@@ -41,6 +42,11 @@ export const routePathToComponent = [
     name: 'EditProfile',
     path: '/edit',
     component: ConnectedEditProfile
+  },
+  {
+    name: 'EditOrg',
+    path: '/edit_organization',
```

Comment:
We'll need to come up with a convention for this. We should probably have it be something like `/{org_id}/edit`

Reply:
that's a good point, we'll need to do the same thing for the professional edit page as well then